range.js

```diff
@@ -40,11 +50,34 @@ exports.range = varArgs(function(specs) {
     });
   });
 
-  
+  var inetAddresses = [];
+  var fromAddress, toAddress, masks;
+  if (errors.length === 0) {
+    parsedSpecs.forEach(function(parsedSpec) {
+      var prefix = parsedSpec[0][0].toString() + '.' + parsedSpec[1][0] + '.';
+       // * in the 4th octet, any variation in the third results in a contiguous block of IPs
+      if (parsedSpec[3][0] === 0 && parsedSpec[3][1] === 255) {
+        fromAddress = prefix + parsedSpec[2][0].toString() + '.0';
+        toAddress = prefix + parsedSpec[2][1].toString() + '.255';
+        masks = ipCalc.calculate(fromAddress, toAddress);
+        Array.prototype.push.apply(inetAddresses, masks.map(mapCalculatorResponseToCIDRNotation));
```

Comment:
I'm curious why we are using `Array.prototype.push.apply` instead of `inetAddresses.push` here